scripts/beachball/customRenderers.ts

```diff
@@ -1,5 +1,5 @@
 import { spawnSync } from 'child_process';
-import GitHubApi from '@octokit/rest';
+import { Octokit as GitHubApi } from '@octokit/rest';
```

Comment:
nit: let's `expose { Octokit as GitHubApi } from '@octokit/rest';` somewhere and `import { GitHubApi } from 'types'` here and other places

Reply:
Doing that feels a bit weird since it's not actually one of our types. I guess I could also just update all the references to say Octokit since that's clearly what they want to be called.